plutus-core/plutus-ir/test/transform/knownCon/list

```diff
@@ -0,0 +1,36 @@
+(let
```

Comment:
Marty is leading the way in us adding comments into test cases explaining what they're testing, let's keep doing that :)